compiler/rustc_attr/src/builtin.rs

```diff
@@ -454,6 +454,13 @@ pub fn find_crate_name(sess: &Session, attrs: &[Attribute]) -> Option<Symbol> {
     sess.first_attr_value_str_by_name(attrs, sym::crate_name)
 }
 
+#[derive(Clone, Debug)]
+pub struct Condition {
+    pub name: (Symbol, Span),
```

Comment:
Could you use two fields (`name` and `name_span`) instead of a tuple here?
That would help to avoid `.0`s and `.1`s.